CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v10.0.0](https://github.com/voxpupuli/puppet-letsencrypt/tree/v10.0.0) (2023-09-27)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-letsencrypt/compare/v9.2.0...v10.0.0)
+
+**Breaking changes:**
+
+- Drop Puppet 6 support [\#318](https://github.com/voxpupuli/puppet-letsencrypt/pull/318) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
did you do a rough check through the module to see if there's something else we can drop in this major release?

Reply:
https://github.com/voxpupuli/puppet-letsencrypt/blob/master/metadata.json#L56

Reply:
should we drop ubuntu 18.04 and debian 10?

Reply:
https://github.com/voxpupuli/puppet-letsencrypt/pull/329

Reply:
Debian 10 still has official LTS, but I have no Jenkins nodes on that, so don't care too much ;-)

Reply:
we don't consider the LTS as supported because it gets limited support.